src/components/network/network-explorer.js

```diff
@@ -80,8 +81,39 @@ const useStyles = makeStyles((theme) => ({
             color: theme.palette.type === 'dark' ? '#000' : '#FFF',
         },
     },
+    selectedTooltipName: {
+        height: (itemSize * 3) / 4,
+        fontSize: '20px',
+        padding: '8px',
+        backgroundColor: theme.palette.type === 'dark' ? '#262626' : '#C8C8C8',
+        '& > span': {
+            width: 230,
+            overflow: 'hidden',
+            whiteSpace: 'nowrap',
+            textOverflow: 'ellipsis',
+        },
+    },
+}));
+
+const useStylesCustomTooltip = makeStyles((theme) => ({
+    arrow: {
+        color: theme.palette.type === 'dark' ? '#404040' : '#f0f0f0',
```

Comment:
We should never do that as if we add a new theme we will need to change this code. 
Instead enrich themes with new variable and use it here